### ISSUE TITLE
Escaping bug in default message and meaning for rules

### ISSUE THREAD

**Issue by Michiel-s** [AmpersandTarski/Ampersand#489]
Consider the following script:

```
CONTEXT Issue489
IN ENGLISH

  RELATION r[B_*B_] [UNI,ASY]

ENDCONTEXT
```
This produces a wrongly escaped default message and meaning in rules.json.
The concept name B_ is escaped as B\\_. For the JSON syntax only double quotes have to be escaped.
This probably is the case for other rules also.
```json
{
    "signals": [],
    "invariants": [
        {
            "origin": "line 4:3, file C:\\Dev\\ampersand\\projects\\Issues\\Issue459.adl",
            "srcConceptId": "B__",
            "ruleAdl": "r~;r |- I[B_]",
            "pairView": null,
            "name": "UNI r[B_]",
            "meaning": "r[B\\_] is univalent",
            "tgtConceptId": "B__",
            "conjunctIds": [
                "conj_0"
            ],
            "message": "Each B\\_ may only have one B\\_ in the relation r"
        },
        {
            "origin": "line 4:3, file C:\\Dev\\ampersand\\projects\\Issues\\Issue459.adl",
            "srcConceptId": "B__",
            "ruleAdl": "r~ /\\ r |- I[B_]",
            "pairView": null,
            "name": "ASY r[B_]",
            "meaning": "r[B\\_] is antisymmetric",
            "tgtConceptId": "B__",
            "conjunctIds": [
                "conj_1"
            ],
            "message": "r[B\\_] is antisymmetric"
        }
    ]
}
```

**Comment by hanjoosten** [comment]
This is not a bug, but a feature. It is a result of the fact that the meaning and message are dumped as ReST (Restructured text). This could easily be changed to another format, supported by Pandoc, like Html, LaTeX etc. To show this, I have changed this default to HTML. 
This format is more obvious, and I guess it is easier to render this properly at the frontend. If you prefer another format, please reopen this issue.

**Comment by Michiel-s** [comment]
After commit 094c7ba the output is as below.

This is even worse. Can it be just plain text? The styling is already covered in the frontend.
```json
{
    "signals": [],
    "invariants": [
        {
            "origin": "line 9:1, file C:\\Dev\\ampersand\\projects\\Issues\\Issue459.adl",
            "tgtConcept": "Boom_tak",
            "ruleAdl": "blad~ /\\ blad |- I[Boom_tak]",
            "pairView": null,
            "name": "ASY blad[Boom_tak]",
            "meaning": "<p>blad[Boom_tak] is antisymmetric</p>",
            "srcConcept": "Boom_tak",
            "conjunctIds": [
                "conj_0"
            ],
            "message": "<p>blad[Boom_tak] is antisymmetric</p>"
        }
    ]
}
```

**Comment by Michiel-s** [comment]
Fixed by #492
